stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/adapter/MessageListItem.kt

```diff
@@ -40,6 +40,7 @@ public sealed class MessageListItem {
         val positions: List<Position> = listOf(),
         val isMine: Boolean = false,
         val messageReadBy: List<ChannelUserRead> = listOf(),
+        val isThreadMode: Boolean = false
```

Comment:
This new parameter in the `MessageItem` constructor is a potentially breaking change. I don't know if any of our clients are creating `MessageItem` instances manually, but it seems possible that they might be. I'd mention this in the changelog.

Reply:
Good point